src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import static frc.robot.Constants.Intake;
+import static frc.robot.Constants.Intake.Ports;
+
+import com.ctre.phoenix.motorcontrol.TalonFXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class IntakeSubsystem extends SubsystemBase implements AutoCloseable {
+
+  /** the lower motor, upper motor follows this one - address this motor */
+  private TalonFX lowerMotor;
```

Comment:
Final declaration on each of these would be nice since they shouldn't be re-initialized, I think?

Reply:
correct, ill resolve it

Reply:
Remembered i did this because mockito cannot inject into final values. I'll add a comment.